File: SevenNightClassPCS/oop-java/m1_7_memorypointers/Pointers.java
class Point {
	public int x,y,z;
}

class NaPoint extends Point {
	
}

public class Pointers {
	
	public static void pointDouble(Point p) {
		int b = p.x*2;
	    p.x = b;
	    p = new Point();
	    p.x = 10;
	}

	public static void main(String[] args) {
		int b = 3;
		Point a = new Point();
		int c = 4;
		a.x = b;
		a.y = c;
		b = 8;
		pointDouble(a);
		System.out.println(b);
		System.out.println(a.x);
	}

}

File: Topics/01_Encapsulation/01_06_Memory_and_pointers/python_source/pointers.py

class Point:
    pass

class NaPoint(Point):
    pass

def pointDouble(p):
    b = p.x*2
    p.x = b
    p = Point()
    p.x = 10
    
b = 3
a = Point()
c = 4
a.x = b
a.y = c
b = 8
pointDouble(a)
print(b)
print(a.x)
